### DISCUSSION TITLE
How to extend new LTMEAT duration?

### DISCUSSION THREAD

**Discussion by KandyKaneMarvel** [3kh0/ext-remover#727] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
New LTMEAT doesn't last very long for me. Maybe 15 minutes. I did find that I could extend it by opening up more windows before flipping the file switch to activate it, but only to an extent. Any help would be appreciated.
Yours truly,
EcoliEater87

**Comment by AshtonDavies** [comment, marked as answer]
Little factors, such as the time the window is closed can cause it to not fully freeze. Try not to close the window right away before the extension tab icons have been established, but don't give them time to load in much either.

The folder capacity greatly ties into that, because it uses more memory, which is helpful as long as it doesn't crash everything.

If you have trouble with this method, the print method of LTMEAT was recently discovered in Titanium Network and it is a good alternative.

**Reply by KandyKaneMarvel** [reply]
Thank you for the reply. I'll have to check that one out after school.
